parsers/to-style-dictionary/tokens/font.ts

```diff
@@ -13,13 +13,12 @@ export class Font extends FontToken {
     return {
       asset: {
         font: (options?.formatTokens?.fontFormat ?? ['ttf']).reduce((acc, format) => {
+          const keys = [...this.keys, format];
```

Comment:
```suggestion
          const path = [...this.keys, format];
```
Rationale: [`setWith(object, path, value)`](https://lodash.com/docs/4.17.15#setWith)